base/applications/regedit/resource.h

```diff
@@ -73,48 +72,19 @@
 
 #define ID_HELP_HELPTOPICS            32794
 #define ID_HELP_ABOUT                 32795
-#define ID_WINDOW_CASCADE             32797
-#define ID_WINDOW_TILE                32798
-#define ID_WINDOW_ARRANGEICONS        32799
-#define ID_OPTIONS_FONT               32800
-#define ID_OPTIONS_AUTOREFRESH        32801
-#define ID_OPTIONS_READONLYMODE       32802
-#define ID_OPTIONS_CONFIRMONDELETE    32803
-#define ID_OPTIONS_SAVESETTINGSONEXIT 32804
-#define ID_SECURITY_PERMISSIONS       32805
-#define ID_VIEW_TREEANDDATA           32806
-#define ID_VIEW_TREEONLY              32807
-#define ID_VIEW_DATAONLY              32808
-#define ID_VIEW_DISPLAYBINARYDATA     32810
-#define ID_VIEW_REFRESHALL            32811
-#define ID_VIEW_REFRESHACTIVE         32812
-#define ID_VIEW_FINDKEY               32813
-#define ID_TREE_EXPANDONELEVEL        32814
 #define ID_TREE_EXPANDBRANCH          32815
-#define ID_TREE_EXPANDALL             32816
 #define ID_TREE_COLLAPSEBRANCH        32817
-#define ID_EDIT_ADDKEY                32818
-#define ID_EDIT_ADDVALUE              32819
-#define ID_EDIT_BINARY                32821
-#define ID_EDIT_STRING                32822
-#define ID_EDIT_DWORD                 32823
-#define ID_EDIT_MULTISTRING           32824
 #define ID_REGISTRY_OPENLOCAL         32825
-#define ID_REGISTRY_CLOSE             32826
 #define ID_REGISTRY_LOADHIVE          32827
 #define ID_REGISTRY_UNLOADHIVE        32828
-#define ID_REGISTRY_RESTORE           32829
-#define ID_REGISTRY_SAVEKEY           32830
 #define ID_REGISTRY_SELECTCOMPUTER    32831
 #define ID_REGISTRY_PRINTSUBTREE      32832
 #define ID_REGISTRY_PRINTERSETUP      32833
```

Comment:
These two (printsubtree and printersetup) are also unused ; they have been replaced by `ID_REGISTRY_PRINT`.
```suggestion
```

Reply:
Ok for ID_REGISTRY_PRINTSUBTREE, But ID_REGISTRY_PRINTERSETUP is still referenced in framwnd.c (with just commented out code. Shall I strip it that commented code there as well?

Reply:
As far as I can see, both these functionalities (the printer setup dialog, and the action of printing the sub-tree) are, in the "new" regedit.exe (Win2000, WinXP/2003, Vista+ and ROS) merged together:
![regedit_winXP_2003_Vista_7](https://github.com/user-attachments/assets/f0d6a706-0da3-4883-8d51-898d1840d153)

Compare it with the original feature of the old-school regedt32 (of which we had a basic clone and from which our regedit has been adapted, see the commits by Rob Dickenson, one of which I pointed to you):

- Printer Setup:
![regedt32_printersetup](https://github.com/user-attachments/assets/97e2822c-c080-4b5d-b623-5161a672e1d6)

- Print Sub-tree:
![regedt32_printsubtree](https://github.com/user-attachments/assets/2bf04753-adbc-40ad-a031-b88912ea00c2)

Reply:
So I think you can remove the code for ID_REGISTRY_PRINTERSETUPin famewnd.c
```c
1341     case ID_REGISTRY_PRINTERSETUP:
1342         //PRINTDLG pd;
1343         //PrintDlg(&pd);
1344         //PAGESETUPDLG psd;
1345         //PageSetupDlg(&psd);
1346         break;
```
Indeed, the `PrintDlg()` is already dealt with in the `case ID_REGISTRY_PRINT: PrintRegistryHive(...);`, and the `PageSetupDlg()` printer page setup is available in the tabs in the print dialog.

Reply:
I acted on your suggestion by https://github.com/reactos/reactos/pull/7194/commits/2f1371a1b2d11429f3f0ef43c74b73b6481c508d
Conversation resolved.